content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -63,20 +63,20 @@ datasource db {
 }
 ```
 
-You can use the Data Proxy with Node.js, or with Edge runtimes like Cloudflare Workers and Vercel Edge Functions (Middleware).
+You can use the Data Proxy with Node.js (deployed in any environment where you can also deploy a normal project using Prisma Client), or with Edge runtimes like Cloudflare Workers and Vercel Edge Functions (Middleware).
 
 #### Node.js
 
-To use the Data Proxy with Node.js, set and define the environment variable as usual. See [Environment Variables](https://www.prisma.io/docs/guides/development-environment/environment-variables).
+To use the Data Proxy with Node.js, set and define the environment variable with your database connection string as usual. See [Environment Variables](https://www.prisma.io/docs/guides/development-environment/environment-variables).
 
-For guidance on how to configure environment variables in your deployment, please refer to your provider's documentation:
+For guidance on how to configure environment variables in your deployment, please refer to your provider's documentation. For example:
```

Comment:
This makes it clearer that AWS and Heroku are just 2 possible providers, there are others with different configuration and documentation.

Reply:
I quite like them, they are def popular ones and it will be easy for us to add more here. I would prefer that.

Reply:
Persisted in https://github.com/prisma/docs/issues/3285